app/helpers/miq_ae_class_helper.rb

```diff
@@ -324,4 +324,26 @@ def datastore_form(ae_ns, sb_data, type)
                             :nameReadOnly  => domain && !ae_ns.editable_property?(:name),
                             :descReadOnly  => domain && !ae_ns.editable_property?(:description)})
   end
+
+  def first_user_summary(ae_method, sb_data)
```

Comment:
What does first_user_summary mean?  I see it also down below in the mode?  I would have expected something like method_details or something as the name.

Reply:
@jeffibm @amalvijayan03 It seems the sample data was using an automate method named FirstUser but that is just a sample...it could be named _anything_, so putting it as the internal method name is not good.

Reply:
Tha changes has been made in the below PR
https://github.com/ManageIQ/manageiq-ui-classic/pull/8913